Tests/MathLib/TestGlobalMatrixInterface.cpp

```diff
@@ -94,24 +94,12 @@ void checkGlobalMatrixInterfaceMPI(T_MATRIX &m, T_VECTOR &v)
 
     MathLib::finalizeMatrixAssembly(m);
 
-    // Multiply by a vector
-    v = 1.;
-    const bool deep_copy = false;
-    T_VECTOR y(v, deep_copy);
-    m.multiply(v, y);
-
-    ASSERT_EQ(sqrt(3*(3*3 + 7*7)), y.getNorm());
```

Comment:
Shall we better keep the test and rewrite it in terms of `BLAS::`? Or shall we remove it entirely?